### DISCUSSION TITLE
sync time

### DISCUSSION THREAD

**Discussion by cracky22** [meshtastic/firmware#4631] [Ideas]
Automatic synchronization of the time even if no real-time clock is installed

**Comment by cracky22** [comment]
> Automatic synchronization of the time even if no real-time clock is installed

Or a function in the Android app that allows you to set / synchronize the time manually

**Comment by smurfy** [comment]
A function like this would be awesome, i have two heltec v3, without gps and one of them is not connected to the internet so cant connect to a timeserver.

If i where able to update the time via app would be great. Especially usefull as @cracky22 mentioned an automatic time sync on app connect or something.

**Comment by cracky22** [comment]
> A function like this would be awesome, i have two heltec v3, without gps and one of them is not connected to the internet so cant connect to a timeserver.
> 
> If i where able to update the time via app would be great. Especially usefull as @cracky22 mentioned an automatic time sync on app connect or something.

Even my Heltec V3 with GPS and RTC does not keep the time...

**Comment by meekowai** [comment]
There is PPS pin on the GPS module which provides a pulse usable for timekeeping so I read. Everyone drops the PPS pin when wiring GPS module.
